LayoutTests/fast/dom/replace-children-mutation-events.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="../../resources/js-test.js"></script>
+<script>
+
+let oldParent = document.createElement('div');
+let span = document.createElement('span');
+oldParent.appendChild(span);
+
+let parent = document.createElement('div');
+parent.innerHTML = '<b>hello</b><i> world</i>';
+let i = parent.querySelector('i');
+span.addEventListener('DOMNodeRemoved', () => i.appendChild(span), {once: true});
+shouldThrowErrorName('parent.replaceChildren(span)', 'NotFoundError');
+
+oldParent.addEventListener('DOMNodeInserted', () => document.body.appendChild(i), {once: true});
+shouldThrowErrorName('span.replaceChildren(oldParent, i, "other")', 'HierarchyRequestError');
```

Comment:
Oh oops, I have a bug here. I'm not testing mutation events because i is span's parent at this point.  Will fix the test as well.